thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -21,6 +21,11 @@ include "rpc.thrift"
 namespace java org.apache.iotdb.confignode.rpc.thrift
 namespace py iotdb.thrift.confignode
 
+struct EndPoint {
```

Comment:
The EndPoint is already defined in rpc.thrift. And I've included it before. Why not just use it?

Reply:
fixed ~